#! /bin/sh -e
## Junichi Uekawa <dancer@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: change call to autoconf213 to autoconf2.13 which exists in Debian
## Updated for 3.7p3 

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
    -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1;;
esac

exit 0

--- canna-3.7p3/mkrelease.sh~	2003-12-28 02:15:20.000000000 +0900
+++ canna-3.7p3/mkrelease.sh	2004-07-02 18:45:06.000000000 +0900
@@ -3,10 +3,10 @@
 set -e
 set -x
 cp Canna.conf.dist Canna.conf
-autoconf259
-autoheader259
+autoconf2.50
+autoheader2.50
 rm -rf autom4te.cache
 cd canuum
-autoconf213
-autoheader259
+autoconf2.13
+autoheader2.50
 rm -rf autom4te.cache
